tslint.json

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "tslint:recommended",
+    "rules": {
+        "indent": "tabs",
```

Comment:
We did; I'm also going to be dumping this file once I've got `resin-tslint` done, so tbh, I'm not hugely bothered. Shall change.